app/models/post.rb

```diff
@@ -14,4 +14,8 @@ def update_current_amount
     self.current_amount = donations.sum(:amount)
     self.save
   end
+
+  def donation_percentage
+    (current_amount / requested_amount) * 100
```

Comment:
if `donation_percentage` is a float you should probably have some check here to ensure the return value is a float.

Reply:
by that I mean, just wrap it and slap a `.to_f` on it